pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/service/PresentationServiceImpl.scala

```diff
@@ -610,6 +596,20 @@ private class PresentationServiceImpl(
     } yield (schemaId, anoncredLibSchema)
   }
 
+  private def resolveCredentialDefinition(
+      credentialId: String
```

Comment:
Should it be named as `credentialDefinitionId`?

Reply:
need to rename it.

Reply:
fixed